app/controllers/jobs_controller.rb

```diff
@@ -6,14 +6,15 @@ class JobsController < ApplicationController
   skip_before_action :authenticate_user!, only: %i[index show]
 
   def index
-    # @jobs = Job.geocoded
-    @markers = @jobs.map do |job|
-      {
-        lat: job.latitude,
-        long: job.longitude,
-        infoWindow: render_to_string(partial: 'info_window', locals: { job: job })
-      }
-    end
+   render_jobs
```

Comment:
this method is already getting called in a `before_action` call so not needed here.